apps/hash-graph/postgres_migrations/V2__ontology_tables.sql

```diff
@@ -1,11 +1,11 @@
 CREATE TABLE IF NOT EXISTS
-  "type_ids" (
-    "version_id" UUID PRIMARY KEY,
+  ontology_ids (
+    ontology_id UUID PRIMARY KEY,
```

Comment:
small nit, does our formatter not ensure quoting/non-quoting for table/column names? I do like to quote column names as it doesn't get strange highlighting overlaps with builtin keywords.

Reply:
The formatter does not really work as expected. I fixed this in https://github.com/hashintel/hash/pull/1975, but I'm happy to do this in this PR as well.

Reply:
No worries, I'm fine with the change being in that PR instead :+1: